src/EditorFeatures/Core/Shared/Extensions/TelemetryExtensions.cs

```diff
@@ -40,7 +40,10 @@ public static short GetScopeIdForTelemetry(this FixAllScope scope)
                 FixAllScope.Document => 1,
                 FixAllScope.Project => 2,
                 FixAllScope.Solution => 3,
-                _ => 4,
+                FixAllScope.Custom => 4,
+                FixAllScope.ContainingMember => 5,
+                FixAllScope.ContainingType => 6,
+                _ => 7,
```

Comment:
hate to nit on this but before we reported `Custom` or `Unknown` as `4`. now we report `Custom` as 4 and `Unknown` as 7. I am fine taking this change but I just need to review out telemetry scripts and make sure I account for this new split. I'll reply back with a :+1: when I am sure we can handle this across past and future telemetry.